### DISCUSSION TITLE
Cannot see the admin view

### DISCUSSION THREAD

**Discussion by carlos-rafael-mbm** [OrchardCMS/OrchardCore#15338] [General]
I have some problems with a project. I downloaded the project for my job and I could execute the app, I can see the website, but when I try to see the admin side, /admin, I can't see this, and in the console it shows a message about this route: Failed to load resource: the server responded with a status of 404 ().
This happens only when I use the project with Visual Studio, because in producttion, it works! Please help me! It is my first task in my new job and my first time with Orchard Core

**Comment by MikeAlhayek** [comment]
which project are you referring you? It seems that your resources are not built which is why you are getting the 404 and the admin is not rendering styles.

**Comment by sebastienros** [comment]
Repro steps please.
